pkg/relayer/proxy/jsonrpc.go

```diff
@@ -78,4 +81,90 @@ func (j *jsonRPCServer) ServiceId() string {
 // when jsonRPCServer is used as an http.Handler with an http.Server.
 // (see https://pkg.go.dev/net/http#Handler)
 func (j *jsonRPCServer) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
+	ctx := request.Context()
+	// Relay the request to the native service and build the response that will be sent back to the client.
```

Comment:
I want us to avoid using the word "native". Consider "supported", "backing", "staked for", etc...